packages/design-system/src/components/Tooltip/Tooltip.tsx

```diff
@@ -212,7 +212,7 @@ export const Tooltip = (props: TooltipProps) => {
       className,
       component,
       dialog,
-      id,
+      // id,
```

Comment:
here's where it gets interesting. 

including `id` in the list of trigger props resulted in `aria-describedby` appearing **only when the user defined an `id`** (`id` returned `undefined` if user didn't define a value, which resulted in `aria-describedby` being removed).

removing `id` from the prop list resulted in the unique `id` and user-created `id` being correctly assigned to `aria-describedby` **but resulted in adding an extra `id` attribute to the tooltip trigger itself** (obviously breaking the rule that every `id` should have a unique value - both button and tooltip have same `id` here).